src/N98/Magento/Command/Developer/Module/DetectComposerDependenciesCommand.php

```diff
@@ -0,0 +1,432 @@
+<?php
+
+declare(strict_types=1);
+
+namespace N98\Magento\Command\Developer\Module;
+
+use Magento\Framework\Component\ComponentRegistrar;
+use Magento\Framework\Component\ComponentRegistrarInterface;
+use Magento\Framework\Filesystem\Io\File;
+use N98\Magento\Command\AbstractMagentoCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+
+class DetectComposerDependenciesCommand extends AbstractMagentoCommand
+{
+    const COMPOSER_FILE_NOT_FOUND = 'COMPOSER_FILE_NOT_FOUND';
+
+    /**
+     * @var ComponentRegistrarInterface
+     */
+    private $componentRegistrar;
+
+    /**
+     * @var array
+     */
+    private $localInstalledPackagesFromLockFile;
+
+    /**
+     * Configure command
+     */
+    protected function configure(): void
+    {
+        $this->setName('dev:module:detect-composer-dependencies')
+            ->addArgument('path', InputArgument::IS_ARRAY, 'Path to modules')
```

Comment:
Hey @cmuench

This is a great new feature, I've just tried it, out of curiosity, but the first time I executed it I got an error that wasn't very helpful:
```
$ n98-magerun2.phar dev:module:detect-composer-dependencies

In Finder.php line 614:

  You must call one of in() or append() methods before iterating over a Finder.
```

This was because I didn't check how to execute the command at first, and because the `path` argument was missing.
It's probably a good idea to mark this argument as required using `InputArgument::IS_ARRAY | InputArgument::REQUIRED`, no?

Thanks! 🙂

Reply:
Yes, good finding. I will change that.